package.json

```diff
@@ -1149,7 +1149,7 @@
     "test": "find ./assets/test -type d -name '.build' -exec rm -rf {} + && find . -type d -name 'Package.resolved' -exec rm -rf {} + && tsc -p ./ && node ./out/test/runTest.js",
     "compile-tests": "find ./assets/test -type d -name '.build' -exec rm -rf {} + && npm run compile && npm run esbuild",
     "package": "vsce package",
-    "dev-package": "vsce package --no-update-package-json 1.9.1-dev",
+    "dev-package": "vsce package --no-update-package-json 1.10.1-dev",
```

Comment:
1.10.0?

Reply:
That's for dev builds. It has to be later than the release build but earlier than what will be the next build.

Reply:
We'd still need to change it for minor version updates though. This is a pretty common pattern in extensions

Reply:
It's a minor thing, I've updated it every release up until now. It'd be good to automate it but didn't find an easy way to do that